packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -112,6 +116,11 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// Specifies whether drag handler is displayed when the drag action is enabled.
+  ///
+  /// Defaults is true in material 3 and false in material 2.
+  final bool? hasDragHandle;
```

Comment:
Some grammar nits, and also adding references:
```suggestion
  /// Specifies whether a drag handle is shown when [enableDrag] is true.
  ///
  /// When null (the default), the behavior is determined by
  /// [BottomSheetThemeData.hasDragHandle].
  final bool? hasDragHandle;
```